tools/autotool.py

```diff
@@ -200,6 +200,10 @@ def get_target(config):
 		platform = config['PLATFORM']
 		target = "arm-helenos"
 
+	if (config['PLATFORM'] == "arm64"):
```

Comment:
Do we really want to (continue to) use this naming convention?
I feel like using arbitrary "our" names instead of conventional ones can only be harmful.

Reply:
Yes. arm64 is not as arbitrary as you might think. For example, Linux, FreeBSD and also Windows are using it. It is generally understood.